services/src/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/model/BlockRequest.java

```diff
@@ -20,21 +20,32 @@
 import lombok.Value;
 
 @Value
-public class InetAddress {
+public class BlockRequest {
+
+    @JsonInclude(Include.NON_NULL)
```

Comment:
I guess `Include.NON_NULL` is needless here
the `containerName` variable can't be null